### DISCUSSION TITLE
Iniciante

### DISCUSSION THREAD

**Discussion by DogCodeSons** [community/community#122377] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Olá, escolhi uma trilha de careira na faculdade e iniciou com Python, instalei o programa no tablet mas não consigo digitar o básico, ele não muda de cor e nem da enter. App Pydroid 3 
"python"
>>> print ("hello world")
nome = input (joao:  ")
print (joao)

**Comment by davevad93** [comment, marked as answer]
Hi @DogCodeSons ,

There's an issue with your code:

```bash
print ("hello world")
nome = input (joao: ")  # Returns a SyntaxError: unterminated string literal (detected at line 2) nome = input (joao: ")
print (joao) # Returns a NameError: name 'joao' is not defined
```
Try this:

```bash
print("hello world")
nome = input("joao:") 
# Output example:
# hello world
# joao: hello (or whatever you write here)
```
Hope this helps!

**Comment by DogCodeSons** [comment]
Hii, 

This work well now. I dont know how can be wrong in my university class.

Thank you 🫶🫶

**Reply by davevad93** [reply]
Hey @DogCodeSons , 

You're welcome, don't worry it's normal to make some mistakes when coding. If you need any assistance or advice with Python feel free to reach out and If I have correctly answered your question in this discussion, mark it as "answered". Have a nice day!
